MobiledgeX/Editor/MobiledgeXEditorWindow.cs

```diff
@@ -350,7 +354,7 @@ void AddMobiledgeXPlugins()
                   }
                   catch (Exception e)
                   {
-                      Debug.LogError(e);
+                      Debug.LogError("MobiledgeX: " + e);
```

Comment:
you can make the "MobiledgeX: " a static string or something, so you don't have to hardcode everytime

Reply:
Will do next update, Thanks for point this out